src/io/io.h

```diff
@@ -7,6 +7,17 @@
 #include "../global/global.h"
 #include "../grid/grid3D.h"
 
+/* Local function that designates whether we are using a root-process. It gives
+ *  * gives a sensible result regardless of whether we are using MPI */
```

Comment:
One formatting suggestion (and fixing a typo -- the typo is leftover from when I originally wrote the docstring)
```suggestion
 *  a sensible result regardless of whether we are using MPI */
```